packages/evm/src/fee/BasicFee.ts

```diff
@@ -6,12 +6,26 @@ import type { EvmFee } from '../types.js';
 
 import type { EvmFeeCalculationParams } from './types.js';
 import { BaseEvmTransferFeeCalculator } from './types.js';
-
+/**
+ * @internal
+ * @category EVM Fee
+ *
+ *
+ * Wrapper class to calculate
+ * fee for a route that uses
+ * basic calculation.
+ */
 export class BasicFeeCalculator extends BaseEvmTransferFeeCalculator {
   constructor() {
     super();
   }
-
+  /**
+   * @category EvmFee
```

Comment:
does category exists as keyword for typedoc?

Reply:
Yes. https://typedoc.org/tags/category/